### PR TITLE
Migration of runtime test languages

### PR DESCRIPTION
**Pull request by svenefftinge**

### CODE REVIEW

File: plugins/org.eclipse.xtend.core/src-gen/org/eclipse/xtend/core/parser/antlr/internal/InternalXtendFlexer.java [hunk 1]
```diff
@@ -56,87 +56,109 @@
     "\1\23\1\103\1\77\1\100\1\102\1\4\1\75\1\73\1\74\1\7"+
     "\1\106\1\107\1\104\1\105\1\112\1\113\1\114\1\115\1\0\41\5"+
     "\2\0\4\4\4\0\1\4\1\33\1\0\1\5\7\0\1\4\4\0"+
-    "\1\4\1\32\4\0\27\4\1\0\37\4\1\0\u013f\4\31\0\162\4"+
```
Comment (szarnekow) [inline plugins/org.eclipse.xtend.core/src-gen/org/eclipse/xtend/core/parser/antlr/internal/InternalXtendFlexer.java]: This looks like different encoding config

File: plugins/org.eclipse.xtext.xbase.ui/plugin.xml_gen [hunk 2]
```diff
@@ -947,7 +947,7 @@
 	        point="org.eclipse.ui.ide.markerResolution">
 	    <markerResolutionGenerator
 	        class="org.eclipse.xtext.xbase.annotations.ui.XbaseWithAnnotationsExecutableExtensionFactory:org.eclipse.xtext.ui.editor.quickfix.MarkerResolutionGenerator"
-	        markerType="org.eclipse.xtext.xbase.annotations.ui.xbasewithannotations.check.fast">
+	        markerType="org.eclipse.xtext.xbase.ui.xbasewithannotations.check.fast">
```
Comment (szarnekow) [inline plugins/org.eclipse.xtext.xbase.ui/plugin.xml_gen]: This should be merged into the plugin.xml

File: tests/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/generator/ecore/parser/antlr/internal/InternalSubTestLanguageParser.java [hunk 3]
```diff
@@ -179,30 +181,30 @@ public final EObject ruleSubMain() throws RecognitionException {
...
-                    // InternalSubTestLanguage.g:108:3: lv_another_3_0= ruleAnotherSuperMain
+                    // InternalSubTestLanguage.g:107:4: (lv_another_3_0= ruleAnotherSuperMain )
+                    // InternalSubTestLanguage.g:108:5: lv_another_3_0= ruleAnotherSuperMain
                     {
-                     
-                    	        newCompositeNode(grammarAccess.getSubMainAccess().getAnotherAnotherSuperMainParserRuleCall_3_0()); 
-                    	    
-                    pushFollow(FollowSets000.FOLLOW_2);
+
+                    					newCompositeNode(grammarAccess.getSubMainAccess().getAnotherAnotherSuperMainParserRuleCall_3_0());
```
Comment (szarnekow) [inline tests/org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/generator/ecore/parser/antlr/internal/InternalSubTestLanguageParser.java]: This isn't nice

File: plugins/org.eclipse.xtend.core/src-gen/org/eclipse/xtend/core/parser/antlr/internal/InternalXtendFlexer.java (hunk 1 above)
Comment (svenefftinge) [inline plugins/org.eclipse.xtend.core/src-gen/org/eclipse/xtend/core/parser/antlr/internal/InternalXtendFlexer.java]: I used Java 1.7. What do you use, when regenerating Xtend?

File: plugins/org.eclipse.xtext.xbase.ui/plugin.xml_gen (hunk 2 above)
Comment (svenefftinge) [inline plugins/org.eclipse.xtext.xbase.ui/plugin.xml_gen]: The plugin.xml for xbase doesn't contain any declarations of issue markers

**Comment by szarnekow** [comment]
I'll revert this PR since it breaks the workspace and master. Also I cannot run `GenerateAllTestLanguages2.mwe2` on my machine (it doesn't finish) but unfortunately I don't have the time to look into it.

**Comment by szarnekow** [comment]
Since two workflows clean the same src-gen folder, the branch caused trouble. I suggest to generate to src-gen-old from the old generator and src-gen from the new one. Also we'd need to add the new generate workflow to the GenerateAll thing that is used by the xtext generator job.

**Pull request merged**